interfaces/builtin/greengrass_support.go

```diff
@@ -250,6 +250,7 @@ mount options=(rw, bind) /proc/sysrq-trigger -> /proc/sysrq-trigger,
 mount options=(rw, bind) /dev/null -> /proc/kcore,
 mount options=(rw, bind) /dev/null -> /proc/sched_debug,
 mount options=(rw, bind) /dev/null -> /proc/timer_stats,
+mount options=(rw, bind) /dev/null -> /proc/latency_stats,
```

Comment:
Please add this comment above your rule:
```
# greengrass will also mount over /proc/latency_stats when running on
# kernels configured with CONFIG_LATENCYTOP set 
```

Reply:
Oh wow! 

This makes so much sense now. :-)

Thanks for the info. It will be helpful for sure.

Reply:
Done!